rpm/Makefile

```diff
@@ -1,11 +1,12 @@
 ARCH=$(shell uname -m)
+ALPINE:=$(shell $(CURDIR)/../detect_alpine_image)
```

Comment:
No It was included as part of: https://github.com/docker/docker-ce-packaging/pull/4